### ISSUE TITLE
Build for Windows

### ISSUE THREAD

**Issue by rojmiller** [hoehermann/purple-gowhatsapp#6]
I found a link to EionRobb's build for Windows in a closed issue. Would it be possible to provide a link to the  Windows build on the main page so it is easier to find?

However, the current build doesn't seem to work for me. The fake conversation pops up, but is blank - there is no QR code to scan....

Windows 10 x64, latest build, Pidgin latest build (2.13.0)

**Comment by hoehermann** [comment]
Thank you for your interest in this plug-in. It has reached a point of "works for me", but that's about it. In my opinion, this plug-in is not mature enough for an official-ish release. This is why I refrain from advertising EoinRobb's windows builds.

I myself do not use Windows. As a result, I can not create, nor test nor debug Windows builds. Also, [compared do Linux](https://developer.pidgin.im/wiki/Installing%20Pidgin#Compiling), [setting up a build system suitable for developing pidgin/purple plugins on Windows](https://developer.pidgin.im/wiki/BuildingWinPidgin) is quite a hassle. I am sorry. I won't fix this.

I would recommend opening an issue at EionRobb's fork, but did not enable issues at his repository. I suspect, he focuses on another project of his.

**Comment by EionRobb** [comment]
I think it's just github defaults for a fork to not have the Issues tab?  I've turned it on now.  I don't know if I'll be able to debug any errors, but I can take a crack at it, @rojmiller

**Comment by rojmiller** [comment]
Thank you EionRobb!

**Comment by hoehermann** [comment]
Discussion related to recent Windows build is available at https://github.com/EionRobb/purple-gowhatsapp/issues/3.

**Comment by hoehermann** [comment]
I tried to prepare a Christmas gift for the Windows folks today. Alas, [mvdan](https://github.com/golang/go/issues/29074#issuecomment-443633670) says 

> You can't cross-compile a package that uses CGO

. It looks like I'd need to set-up a virtual machine to overcome this limitation. I do not want to do so and I do not even have a server with the necessary capabilities. This means, I cannot provide cross-compiled nightly builds.

Until something changes, I want to keep https://github.com/hoehermann/purple-gowhatsapp/wiki/Windows-Build updated and pointing to the most recend Windows binary I know of.

**Comment by hoehermann** [comment]
Windows users rejoice for I bring good news!

A few weeks ago while working on the [win32 variant of purple-signal](https://github.com/hoehermann/purple-signal/tree/win32), I realized I do not actually need to satisfy all build dependencies listed. They are only needed when developing Pidgin itself. For developing against libpurple, a way smaller subset of libraries is sufficient.

Still not having a Windows vm for nightly builds, it occurred to me to give wine a shot. Wine is mature enough. The go compiler works! Just yesterday I succeeded in the first win32 build of this plug-in. :) Go and grab it at https://buildbot.hehoe.de/purple-gowhatsapp/builds/.

Issue resolved! :)
